app/src/renderer/components/governance/ModalDeposit.vue

```diff
@@ -25,18 +25,11 @@
         type="text"
         readonly="readonly"
       />
-      <tm-field
-        v-focus
-        id="amount"
-        :max="balance"
-        :min="0"
-        v-model="amount"
-        type="number"
-      />
+      <tm-field v-focus id="amount" :min="0" v-model="amount" type="number" />
```

Comment:
Removing the :max here I think will remove the limit on the input field, then the user won't know why the button is disabled (although it should be pretty obvious)

Reply:
but then it shows the error message

Reply:
True, I haven't thought of that